apps/src/storage/dataBrowser/dataUtils.js

```diff
@@ -118,8 +118,11 @@ export function editableValue(val) {
  * @returns {string}
  */
 export function displayableValue(val) {
-  if (val === null || val === undefined || val === '') {
-    return '';
+  if (val === null) {
+    return 'null';
+  } else if (val === undefined) {
+    return 'undefined';
+  } else {
```

Comment:
I think JSON.Stringify('') returns "''" (hard to see that but it quotes the quote) - is that intended here?

Reply:
For now, I think yes. Lots of open questions still about how to correctly parse user input.
But just to keep things moving for dataviz, I think we can punt on some of those questions for now.